pkg/dnsprovider/providers/google/clouddns/internal/stubs/managed_zone.go

```diff
@@ -22,10 +22,9 @@ import "k8s.io/federation/pkg/dnsprovider/providers/google/clouddns/internal/int
 var _ interfaces.ManagedZone = ManagedZone{}
 
 type ManagedZone struct {
-	Service *ManagedZonesService
-	Name_   string
-	Id_     uint64
-	Rrsets  []ResourceRecordSet
+	Name_  string
```

Comment:
[Clarification]: Am a little confused with the wordings of the commit and then the code; do u mean that u are making some of the methods re entrant?